Sources/App/Views/PackageController/PackageReleases+View.swift

```diff
@@ -0,0 +1,40 @@
+import Vapor
+import Plot
+
+enum PackageReleases {
+    
+    class View: TurboFrame {
+        
+        let model: Model?
+
+        init(model: Model?) {
+            self.model = model
+            super.init()
+        }
+
+        override func frameIdentifier() -> String {
+            "releases"
+        }
+
+        override func frameContent() -> Node<HTML.BodyContext> {
+            guard let releases = model?.releases
+            else { return .empty }
```

Comment:
![Captured 2021-07-05 at 4 56 21](https://user-images.githubusercontent.com/15193942/124496890-f38d3680-ddb1-11eb-9727-19773398e363.png)

Unfortunately, some packages just return 0 releases (the repo has tags, just not releases).

This gets caught here in this guard, we'll want to return some form of empty state here but no idea what it would look like.

Reply:
> ![Captured 2021-07-05 at 4 56 21](https://user-images.githubusercontent.com/15193942/124496890-f38d3680-ddb1-11eb-9727-19773398e363.png)
> 
> Unfortunately, some packages just return 0 releases (the repo has tags, just not releases).
> 
> This gets caught here in this guard, we'll want to return some form of empty state here but no idea what it would look like.

I think this shouldn't necessarily be a "Releases" tab, but "Release Notes" instead. I'm not sure I'd stick just a standard "Releases" tab down here with the README. We kinda cover "Releases" up in the top section with the significant versions.

So, with that distinction. I think we just have content saying "This package has no release notes".